signing_service/signing_service/signing_service.py

```diff
@@ -196,25 +370,55 @@ def __call__(self, parser, namespace, values, option_string=None):
     ethereum_private_key_parser_group.add_argument(
         '--ethereum-private-key',
         dest='ethereum_private_key',
-        action=make_secret_provider_factory(read_command_line=True, base64_convert=True),
+        action=make_secret_provider_factory(read_command_line=True),
         help='Ethereum private key for Singing Service.',
     )
     ethereum_private_key_parser_group.add_argument(
         '--ethereum-private-key-path',
         dest='ethereum_private_key',
-        action=make_secret_provider_factory(use_file=True, base64_convert=True),
+        action=make_secret_provider_factory(use_file=True),
         help='Ethereum private key for Singing Service.',
     )
     ethereum_private_key_parser_group.add_argument(
         '--ethereum-private-key-from-env',
         dest='ethereum_private_key',
-        action=make_secret_provider_factory(env_variable_name='ETHEREUM_PRIVATE_KEY', base64_convert=True),
+        action=make_secret_provider_factory(env_variable_name='ETHEREUM_PRIVATE_KEY'),
         help='Ethereum private key for Singing Service.',
     )
-    parser.set_defaults(ethereum_private_key=b64decode(os.environ.get('ETHEREUM_PRIVATE_KEY')) if os.environ.get('ETHEREUM_PRIVATE_KEY') is not None else None)  # type: ignore
+    parser.set_defaults(
+        ethereum_private_key=os.environ.get('ETHEREUM_PRIVATE_KEY')
```

Comment:
conditional as default doesn't seem right... especially, that `None` seem to be a bad default value for a private key ;)

Reply:
Removed along with changes from https://github.com/golemfactory/concent/pull/688.